LBH.AdultSocialCare.Api/V1/Controllers/HomeCare/HomeCarePackageController.cs

```diff
@@ -103,6 +102,25 @@ public class HomeCarePackageController : BaseController
             }
         }
 
+        /// <summary>Get all Home Care Packages</summary>
+        /// <returns>The list of Home Care Package Response model</returns>
+        [ProducesResponseType(typeof(IList<HomeCarePackageResponse>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesDefaultResponseType]
+        [HttpGet]
+        [Route("getAll")]
+        public async Task<ActionResult<IList<HomeCarePackage>>> GetAll()
+        {
+            try
+            {
+                IList<HomeCarePackage> result = await _getAllHomeCarePackageUseCase.GetAllAsync().ConfigureAwait(false);
```

Comment:
Should we be converting to response object here as the use case should return a domain object?

Reply:
@dunnudge use case should return domain objects